core/src/main/scala/com/microsoft/azure/synapse/ml/explainers/ICEFeature.scala

```diff
@@ -53,6 +54,14 @@ object ICECategoricalFeature {
 
     ICECategoricalFeature(name, numTopValues, outputColName)
   }
+
+  def fromScalaMap(inputMap: Map[String, Any]): ICECategoricalFeature = {
```

Comment:
R does not let you grab a java hash map.  Even if you call a scala method that returns one, by the time it gets back to R, the deserialization turns it into a normal R environment. But this works well.